### PR TITLE
fix(example-deployment): fix reporting infinite redirect

### PR DESCRIPTION
**Pull request by koonpeng**
## What's new

The root cause is because `/etc/nginx/conf.d/default.conf` already contains a `location / {}` block and that conflicts with the declaration in `/etc/nginx/conf.d/reporting.conf`. The reason it works in dashboard is because `dashboard.conf` gets loaded before `default.conf` because or file system sorting ("da" comes before "de").

This also adds the admin user for use with the new authorization changes. It should be used in favor of "example" when performing some things like submitting tasks.

## Self-checks

- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: example-deployment/docker/dashboard.dockerfile [hunk 1]
```diff
@@ -19,9 +19,17 @@ FROM nginx:stable
 COPY --from=0 /root/rmf-web/packages/dashboard/build /usr/share/nginx/html/dashboard
 SHELL ["bash", "-c"]
 RUN echo -e 'server {\n\
+  listen 80;\n\
+  server_name localhost;\n\
```
Comment (cnboonhan) [inline example-deployment/docker/dashboard.dockerfile:23]: should this be example.com, in the example-deployment context?

Comment (cnboonhan) [inline example-deployment/docker/dashboard.dockerfile:23]: ( i accidentally deleted the previous comment sorry ) should this be example.com, in the example-deployment context?

Comment (cnboonhan) [reply example-deployment/docker/dashboard.dockerfile:23]: cancel the above, i think its incorrect. much to learn!
